build.js

```diff
@@ -37,7 +37,7 @@ const moduleCode = standaloneCode(ajv, {
 });
 
 // Now you can write the module code to file
-fs.writeFileSync(path.join(__dirname, '/src/validations.mjs'), moduleCode);
+fs.writeFileSync(path.join(__dirname, '/src/validations.js'), moduleCode);
```

Comment:
.mjs clashing in unit tests - shouldn't matter in the final build of the contract